### PR TITLE
Resolve #954, #975, #976, #973, #980, #968, #977, #979

### LINKED ISSUE #954: Standalone receipt page
url: /receipt?[transferId=|receivingId=]

The page should handle api calls by itself

Common:

```
sender
destination
transferAmount
send/receive/cancelTime
```

If logged in:
For sender:

```
send/cancelTxHash
```

For receiver:

```
receiveTxHash
```

design: https://www.figma.com/file/XTQwnQ8SZGGDYo2w5nFqH9/Chainsfr_v1.3?node-id=7%3A581

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -164,6 +164,9 @@ class App extends Component {
                   path={`${paths.receipt}`}
                   component={userIsAuthenticated(ReceiptContainer)}
                 />
+                <Route path='*'>
+                  <> Fuck </>
```
Comment (InVincible2016) [inline src/App.js]: remove

File: src/reducers/transferReducer.js [hunk 2]
```diff
@@ -28,6 +28,11 @@ export default function (state = initialState, action) {
         ...state,
         transfer: action.payload.transferData
       }
+    case 'GET_TRANSFER_PASSWORD_FULFILLED':
```
Comment (InVincible2016) [inline src/reducers/transferReducer.js]: These changes should be included in another issue.

File: src/transferStates.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const transferStates = {
```
Comment (InVincible2016) [inline src/transferStates.js]: These changes should be included in another issue.

File: src/actions/transferActions.js [hunk 4]
```diff
@@ -17,19 +17,7 @@ import type { AccountData } from '../types/account.flow.js'
...
-  SEND_CONFIRMED_RECEIVE_PENDING: 'SEND_CONFIRMED_RECEIVE_PENDING',
-  SEND_CONFIRMED_RECEIVE_FAILURE: 'SEND_CONFIRMED_RECEIVE_FAILURE',
-  SEND_CONFIRMED_RECEIVE_CONFIRMED: 'SEND_CONFIRMED_RECEIVE_CONFIRMED',
-  SEND_CONFIRMED_RECEIVE_NOT_INITIATED: 'SEND_CONFIRMED_RECEIVE_NOT_INITIATED',
-  SEND_CONFIRMED_RECEIVE_EXPIRED: 'SEND_CONFIRMED_RECEIVE_EXPIRED',
-  SEND_CONFIRMED_CANCEL_PENDING: 'SEND_CONFIRMED_CANCEL_PENDING',
-  SEND_CONFIRMED_CANCEL_CONFIRMED: 'SEND_CONFIRMED_CANCEL_CONFIRMED',
-  SEND_CONFIRMED_CANCEL_FAILURE: 'SEND_CONFIRMED_CANCEL_FAILURE'
-}
+import transferStates from '../transferStates'
```
Comment (InVincible2016) [inline src/actions/transferActions.js]: These changes should be included in another issue.

File: src/actions/transferActions.js [hunk 5]
```diff
@@ -552,6 +550,15 @@ async function _getTransferHistory (offset: number = 0) {
   return { hasMore, transferData, offset }
 }
 
+async function _getTransferPassword (transferId: string): Promise<string> {
```
Comment (InVincible2016) [inline src/actions/transferActions.js]: These changes should be included in another issue.

File: src/actions/transferActions.js [hunk 6]
```diff
@@ -633,6 +640,13 @@ function getTransferHistory (offset: number) {
   }
 }
 
+function getTransferPassword (transferId: string) {
```
Comment (InVincible2016) [inline src/actions/transferActions.js]: These changes should be included in another issue.

File: src/components/AccountDropdownComponent.jsx [hunk 7]
```diff
@@ -134,17 +134,25 @@ class AccountDropdownComponent extends Component<Props, State> {
...
-                  <Typography className={classes.securityAnswerBtnHelperText}>
-                    {value.cryptoType === 'bitcoin'
-                      ? `${value.hdWalletVariables.xpub.slice(
-                          0,
-                          16
-                        )}...${value.hdWalletVariables.xpub.slice(-24)}`
-                      : value.address}
-                  </Typography>
-                </div>
+                <Grid container direction='row' alignItems='center'>
```
Comment (InVincible2016) [inline src/components/AccountDropdownComponent.jsx]: These changes should be included in another issue.

**Comment by timothywangdev** [comment]
@InVincible2016 https://www.figma.com/file/XTQwnQ8SZGGDYo2w5nFqH9/Chainsfr_v1.3?node-id=0%3A1

See sender receipt design, all pages have security answers. Security answer on cancellation receipt might possibly be reused, thus must be shown.

**Pull request merged**